doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
+
+You can also use a Unix domain socket (``unix/:``) instead of ``host:port``.
+
+.. NOTE::
+
+    The ``host`` value cannot be ``0.0.0.0``.
+
+
+.. _configuration_reference_iproto_advertise_client:
+
+.. confval:: iproto.advertise.client
+
+    An URI used to advertise the current instance to clients.
+
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_IPROTO_ADVERTISE_CLIENT
+
+.. _configuration_reference_iproto_advertise_peer:
+
+.. confval:: iproto.advertise.peer
+
+    An URI used to advertise the current instance to other cluster members.
+    For example, it can be used by an instance to connect to another instance as a replica.
+
+    **Example**
+
+    In the example below, the following configuration options are specified:
+
+    -   In the :ref:`credentials <configuration_reference_credentials>` section, the ``replicator`` user with the ``replication`` role is created.
+    -   ``iproto.advertise.peer`` specifies that other instances should connect to an address defined in :ref:`iproto.listen <configuration_reference_iproto_listen>` using the ``replicator`` user.
+
+    ..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/auto_leader/config.yaml
+        :language: yaml
+        :start-at: credentials:
+        :end-at: peer: replicator@
+        :dedent:
+
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_IPROTO_ADVERTISE_PEER
+
+.. _configuration_reference_iproto_advertise_sharding:
+
+.. confval:: iproto.advertise.sharding
+
+    An URI used by a router and rebalancer.
+
+    **Example**
+
+    In the example below, the following configuration options are specified:
+
+    -   In the :ref:`credentials <configuration_reference_credentials>` section, the ``replicator`` and ``storage`` users are created.
+    -   ``iproto.advertise.peer`` specifies that other instances should connect to an address defined in :ref:`iproto.listen <configuration_reference_iproto_listen>` with the ``replicator`` user.
+    -   ``iproto.advertise.sharding`` specifies that a router should connect to storages using an address defined in :ref:`iproto.listen <configuration_reference_iproto_listen>` with the ``storage`` user.
+
+    ..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/config.yaml
+        :language: yaml
+        :start-at: credentials:
+        :end-at: sharding: storage@
+        :dedent:
+
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_IPROTO_ADVERTISE_SHARDING
+
+
+.. _configuration_reference_iproto_listen:
+
+.. confval:: iproto.listen
+
+    An address used to listen for incoming requests.
+    This address is used for different purposes, for example:
+
+    -   Communicating between cluster members.
+    -   Connecting to an instance using an :ref:`admin console <admin-security>`.
```

Comment:
The admin console is a separate concept. First, it works by another protocol (text, not binary). Next, it is usually local (a unix socket). It is configured by another option: `console.socket`.

Reply:
Well, `tt connect` can work using the binary protocol (over iproto), it creates some confusion.

Reply:
Changed to:
```
Connecting to an instance remotely using an :ref:`admin console <admin-security>`.
```

Reply:
I still feel it as an improper term. The link (admin-security page) makes a clear difference:

1. Admin console. Text protocol, no password, all the commands are executed from `admin` user. Usually a Unix socket.
2. Iproto socket. Binary protocol, authorization is required (except for `guest` user). Inet or Unix socket.

`tt connect` can work using any of the protocols, but doesn't make the iproto socket being an admin console. It just means that the iproto socket can be used for administration purposes.

If we go to strictier terms, I would also rephrase the following paragraph.

> This address is used for different purposes, for example:
>
> * Communicating between cluster members.
> * Connecting to an instance remotely using an [admin console].
> * Connecting to an instance using [connectors] for different languages.

Look, the address itself is used for two purposes:

* Bind a listening socket for the binary protocol.
* Enrich `iproto.advertise.{peer,sharding}` if it is in form of `user@` or `user:password@` or replace if it is missed.

However, the listening socket itself can be used for all the mentioned purposes:

* Communicate between replicaset peers or cluster members.
* Connect from external services or tools.
* Remote administration using `tt connect`.

Reply:
Okay, now I got what the `admin console` means. I meant exactly `remote administration`, thanks for your clarification.

Updated the list as follows:
<img width="820" alt="image" src="https://github.com/tarantool/doc/assets/38073144/7785a72b-34f3-43c6-a405-f9d9b9fadd43">

Reply:
Looks OK, thanks!